floss/main.py

```diff
@@ -42,6 +43,16 @@
 
 MIN_STRING_LENGTH_DEFAULT = 4
 
+WINDOWS_FILES = ['KERNEL32.DLL', 'GDI32.DLL', 'USER32.DLL', 'COMCTL32.DLL',
+                 'COMDLG32.DLL', 'WS2_32.DLL', 'ADVAPI32.DLL', 'NETAPI32.DLL',
+                 'OLE32.DLL', 'SHELL32.DLL', 'WUSER32.DLL', 'PSAPI.DLL',
+                 'MSVCRT.DLL', 'CRTDLL.DLL', 'MSCOREE.DLL']
```

Comment:
this is a good initial list. how did you pick these? should we add a comment saying "These are referenced in PMA" or similar?

Reply:
this should be a Set rather than List

Reply:
@williballenthin 
> this is a good initial list. how did you pick these? should we add a comment saying "These are referenced in PMA" or similar?

Wikipedia + some that tend to appear in some files I tried this with. As said in the PR description:
> The list of Windows files is also not complete. I am not sure if we can get a complete list of files somewhere.